content/sensu-go/6.8/operations/deploy-sensu/secure-sensu.md

```diff
@@ -249,6 +249,130 @@ However, deployments can also use the same certificates and keys for etcd peer a
 
 The Sensu backend checks certificate revocation list (CRL) and Online Certificate Status Protocol (OCSP) endpoints for agent mTLS, etcd client, and etcd peer connections whose remote sides present X.509 certificates that provide CRL and OCSP revocation information.
 
+## Optional: Configure Sensu for FIPS compliance
+
+Sensu provides a Linux amd64 OpenSSL-linked build that supports the Federal Information Processing Standard (FIPS) for Federal Risk and Authorization Management Program (FedRAMP) compliance.
+
+The Sensu build with FIPS-mode configuration options is linked with the FIPS 140-2 validated cryptographic library.
+Sensu builds comply with the FIPS-mode kernel option to enforce FIPS systemwide in Red Hat Enterprise Linux (RHEL).
+[Contact Sensu][13] to request the build with FIPS support.
+
+On systems with FIPS kernel mode, Sensu backends and agents will initialize if the `require-fips` and `require-openssl` configuration options are set to true in the backend and agent configuration files.
+
+Sensu backends and agents on systems with FIPS kernel mode will connect to PostgreSQL running in FIPS mode.
+If PostgreSQL is not running in FIPS mode, Sensu backends and agents on systems with FIPS kernel mode will *not* connect to PostgreSQL.
```

Comment:
After re-reading my notes I think Sensu backends can connect another server running PostgreSQL so long as the SSL handshake succeeds. I've updated the GH issue with the change and moved the existing content to #6.

Reply:
Thank you! I updated the doc according to your clarifications.